common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -32,33 +33,6 @@ namespace rviz_plugins
 {
 AutowareStatePanel::AutowareStatePanel(QWidget * parent) : rviz_common::Panel(parent)
 {
-  // Gate Mode
-  auto * gate_prefix_label_ptr = new QLabel("GATE: ");
-  gate_prefix_label_ptr->setAlignment(Qt::AlignRight);
-  gate_mode_label_ptr_ = new QLabel("INIT");
-  gate_mode_label_ptr_->setAlignment(Qt::AlignCenter);
-  auto * gate_layout = new QHBoxLayout;
-  gate_layout->addWidget(gate_prefix_label_ptr);
-  gate_layout->addWidget(gate_mode_label_ptr_);
-
-  // Selector Mode
-  auto * selector_prefix_label_ptr = new QLabel("SELECT: ");
-  selector_prefix_label_ptr->setAlignment(Qt::AlignRight);
-  selector_mode_label_ptr_ = new QLabel("INIT");
-  selector_mode_label_ptr_->setAlignment(Qt::AlignCenter);
-  auto * selector_layout = new QHBoxLayout;
-  selector_layout->addWidget(selector_prefix_label_ptr);
-  selector_layout->addWidget(selector_mode_label_ptr_);
-
-  // State
-  auto * state_prefix_label_ptr = new QLabel("STATE: ");
-  state_prefix_label_ptr->setAlignment(Qt::AlignRight);
-  autoware_state_label_ptr_ = new QLabel("INIT");
-  autoware_state_label_ptr_->setAlignment(Qt::AlignCenter);
-  auto * state_layout = new QHBoxLayout;
-  state_layout->addWidget(state_prefix_label_ptr);
-  state_layout->addWidget(autoware_state_label_ptr_);
-
   // Gear
   auto * gear_prefix_label_ptr = new QLabel("GEAR: ");
   gear_prefix_label_ptr->setAlignment(Qt::AlignRight);
```

Comment:
@h-ohta 
can you fix this indent?

Reply:
Hmm...it looks no problem in https://github.com/autowarefoundation/autoware.universe/blob/86646cb0642993ddba37f109e9578cc6063ca2fd/common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp.

Reply:
is there tabs and space?
![image](https://user-images.githubusercontent.com/65527974/205209247-7b17e61d-2561-4272-9c8c-baf14e20c24c.png)

Reply:
In my editor, only tabs exist.
![Uploading image.png…]()

Reply:
it looks like this is because of github' function
![image](https://user-images.githubusercontent.com/65527974/205209962-92af17bf-4597-4028-9199-241148860b9b.png)

Reply:
And if there are some indents, pre-commit can fix it.